datalad_registry/datasets.py

```diff
@@ -1,15 +1,19 @@
-"""Blueprint for /datasets view
+"""Blueprint for root view
 """
```

Comment:
wow github gives me an odd diff here

![image](https://user-images.githubusercontent.com/39889/131737377-37baa915-dcaf-4d91-bf5c-a08cfefa6257.png)

Reply:
I see that too.  No idea why.